libraries/AP_BoardConfig/px4_drivers.cpp

```diff
@@ -518,7 +537,7 @@ void AP_BoardConfig::px4_sensor_error(const char *reason)
     */
     while (true) {
         printf("Sensor failure: %s\n", reason);
-        hal.console->printf("Sensor failure: %s\n", reason);
+        hal.console->printf("Check BRD_TYPE: %s\n", reason);
```

Comment:
the problem is the 50 character limit for a STATUSTEXT (its now a STATUSTEXT so that it displays on the HUD in MP)